PIC_database_updater.py

```diff
@@ -139,26 +189,81 @@ def update_elevation(self, barcode, max_elev, min_elev, elev_unit):
 
         locality_id = self.get_locality_id_with_collectingevent(collecting_event_id=collecting_event_id)
 
-        condition = f"""WHERE LocalityID = {locality_id};"""
+        is_present = self.sql_csv_tools.get_one_match(tab_name="locality", id_col="MaxElevation", key_col="LocalityID",
+                                                      match=f"{locality_id}", match_type=int)
+
+        if pd.isna(is_present) or self.force_update:
+
+            condition = f"""WHERE LocalityID = {locality_id};"""
+
+            sql = self.sql_csv_tools.create_update_statement(tab_name='locality', col_list=['MaxElevation', 
+                                                                                            'MinElevation',
+                                                                                            'OriginalElevationUnit'],
+                                                             val_list=[max_elev, min_elev, elev_unit],
+                                                             condition=condition
+                                                             )
+
+            self.sql_csv_tools.insert_table_record(sql=sql)
+
+        else:
+            self.logger.info(f"Elevation already in locality table at: {locality_id}")
 
```

Comment:
locality detail tab is not already created for many barcodes, so this function will create the locality detail record and link it to the locality id, in the cases where the locality detail record is already , it will simply be updated with UTM or TNRS instead.